CMakeLists.txt

```diff
@@ -2,5 +2,6 @@
 # executed from 'project/build' with 'cmake ../'.
 cmake_minimum_required(VERSION 2.6)
 find_package(Rock)
+add_definitions(-fext-numeric-literals)
```

Comment:
Alright ... so this PR becomes obsolete if we fix `base/cmake` ? Shouldn't we just close it then (since the revert got merged and we're going to fix the gnu++11 thingy) ?